docroot/modules/custom/foia_webform/src/FoiaSubmissionServiceApi.php

```diff
@@ -122,7 +122,8 @@ protected function assembleRequestData(FoiaRequestInterface $foiaRequest) {
 
     $apiVersion = ['version' => FoiaSubmissionServiceInterface::VERSION];
     $foiaRequestId = ['request_id' => $foiaRequest->id()];
-    $submissionValues = array_merge($foiaRequestId, $apiVersion, $agencyInfo, $formValues);
+    $confirmationId = ['confirmation_id' => $foiaRequest->get('field_webform_submission_id')->value];
```

Comment:
On line 118 we have:
```
$formValues = $this->getSubmissionValues($foiaRequest);
```

Within that method (`getSubmissionValues`), we grab the value for the submission ID so that we can load the webform submission. I wonder if we could just modify that method to include the `confirmation_id` as part of the array that `getSubmissionValues` returns so that we don't have to fetch the webform submission ID twice.